parquet-column/src/test/java/org/apache/parquet/schema/TestPrimitiveComparator.java

```diff
@@ -186,22 +186,22 @@ public void testDoubleComparator() {
   @Test
   public void testLexicographicalBinaryComparator() {
     testObjectComparator(LEXICOGRAPHICAL_BINARY_COMPARATOR,
-      Binary.fromConstantByteArray(new byte[0]),
-      Binary.fromConstantByteArray(new byte[]{127, 127, 0, 127}, 2, 1),
-      Binary.fromCharSequence("aaa"),
-      Binary.fromString("aaaa"),
-      Binary.fromReusedByteArray("aaab".getBytes()),
-      Binary.fromReusedByteArray("azzza".getBytes(), 1, 3),
-      Binary.fromReusedByteBuffer(ByteBuffer.wrap("zzzzzz".getBytes())),
-      Binary.fromReusedByteBuffer(ByteBuffer.wrap("aazzzzzzaa".getBytes(), 2, 7)),
-      Binary.fromConstantByteBuffer(ByteBuffer.wrap(new byte[]{-128, -128, -128})),
-      Binary.fromConstantByteBuffer(ByteBuffer.wrap(new byte[]{-128, -128, -1}, 1, 2))
+      Binary.fromConstantByteArray(new byte[0]),                                        // ||
+      Binary.fromConstantByteArray(new byte[]{127, 127, 0, 127}, 2, 1),                 // |00|
+      Binary.fromCharSequence("aaa"),                                                   // |61|61|61|
+      Binary.fromString("aaaa"),                                                        // |61|61|61|61|
+      Binary.fromReusedByteArray("aaab".getBytes()),                                    // |61|61|61|62|
+      Binary.fromReusedByteArray("azzza".getBytes(), 1, 3),                             // |7A|7A|7A|
+      Binary.fromReusedByteBuffer(ByteBuffer.wrap("zzzzzz".getBytes())),                // |7A|7A|7A|7A|7A|7A|
+      Binary.fromReusedByteBuffer(ByteBuffer.wrap("aazzzzzzaa".getBytes(), 2, 7)),      // |7A|7A|7A|7A|7A|7A|61|
+      Binary.fromConstantByteBuffer(ByteBuffer.wrap(new byte[]{-128, -128, -128})),     // |80|80|80|
+      Binary.fromConstantByteBuffer(ByteBuffer.wrap(new byte[]{-128, -128, -1}, 1, 2))  // |80|FF|
```

Comment:
Thanks for writing these comment, although it would have been enough for the byte[] values. From this particular line, however, one 80 is missing.

Reply:
Here we are creating a `Binary` object with offset `1` and length `2` so no `80` is missing.